packages/office-ui-fabric-react/src/components/Fabric/Fabric.base.tsx

```diff
@@ -29,7 +29,26 @@ export class FabricBase extends React.Component<
     });
     const divProps = getNativeProps<React.HTMLAttributes<HTMLDivElement>>(this.props, divProperties);
 
-    return <div {...divProps} className={classNames.root} ref={this._rootElement} />;
+    let fabricComponent;
+    if (this.props.applyTheme) {
+      // apply theme only to the div
+      fabricComponent = (
+        <div
+          {...divProps}
+          style={{ backgroundColor: this.props.theme!.semanticColors.bodyBackground }}
```

Comment:
please can we do this inside of the .styles file? Also, you're not applying the text color.

Reply:
done.